README.md

```diff
@@ -1,43 +1,50 @@
-# Charybdis Wireless (3x6 aka Mini) Guide
 ![wireless charybdis](./images/DSC08373.jpg "My build")
 
-#### Table of contents
-* [Before we begin](#before-we-begin)
-* [Disclaimer](#disclaimer)
-* [Build Guide](#build-guide)
-    * [1. Required Parts](#step-1-required-parts)
-    * [2. Assembly](#step-2-assembly)
-    * [3. Software](#step-3-software)
-* [Thanks](#thanks)
-
+# Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Before we begin](#before-we-begin)
+- [Disclaimer](#disclaimer)
+- [Required Parts](#required-parts)
+  - [Flexible PCBs](#flexible-pcbs)
+  - [PMW3610 Breakout](#pmw3610-breakout)
+  - [Nice!Nano Holder](#nicenano-holder)
+  - [3D Prints](#3d-prints)
+    - [Observations after build](#observations-after-build)
+  - [Electronic components](#electronic-components)
+    - [Notes](#notes)
+  - [Helper Tools](#helper-tools)
+- [Assembly](#assembly)
+- [Software](#software)
+- [Thanks](#thanks)
 # Before we begin
 
 The Charybdis was created and designed by [Bastard Keyboards](https://bastardkb.com/) so I take no credit in the creation or design. To learn more about the Charybdis itself, read about it on [Charybdis Github](https://github.com/Bastardkb/Charybdis) repo. 
 
 The purpose of this guide is mainly to take a note on what I have done to building a Wireless (Bluetooth) Charybdis, since there is almost no written guide when it comes to building a Wireless Charybdis. I am going to build a 3x6 Charybdis Mini, therefore my guide will be specifically for that. If you are planning to build Charybdis Nano (or add a rotary knob), etc.. Please fork and tweak it as you like (with reference please).
 
 **Important notes:**
-- As of writing this setup does not support RGB LEDs.
-
-- The necessary files for PCBs and 3D files are scattered across Github, I will be referring to the each file specifically in this guide.
+- As of writing this setup does not support RGB LEDs
+- The necessary files for PCBs and 3D files are scattered across Github, I will be referring to each file specifically in this guide
 
 # Disclaimer 
-**Follow at your own risk**, I am not liable for anything that does not work. If you are unsure of something I would suggest you stop by the Bastard Keyboards discord if you have questions. 
+**Follow at your own risk**, I am not liable for anything that does not work. If you are unsure of something I would suggest you stop by the Bastard Keyboards discord if you have questions. Please note, as it's not an official supported build you will also get limited help there.
```

Comment:
I believe we can set a positive toning here by letting them people can get "community help" instead of "limited help" because its not officially supported.